projects/extension/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+
+## Added
+
+- Add extension logging strategy (#429](https://github.com/paritytech/substrate-connect/issues/429))
```

Comment:
```suggestion
- Add a new tab showing the logs of the underlying client (#429](https://github.com/paritytech/substrate-connect/issues/429))
```

"Add extension logging strategy" doesn't mean anything from the point of view of a user